beard/applications/learning_model.py

```diff
@@ -0,0 +1,221 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of Beard.
+# Copyright (C) 2015 CERN.
+#
+# Beard is a free software; you can redistribute it and/or modify it
+# under the terms of the Revised BSD License; see LICENSE file for
+# more details.
+
+"""Tools for learning the distance model."""
+
+import cPickle
+import json
+import numpy as np
+
+from sklearn.ensemble import GradientBoostingClassifier
+from sklearn.feature_extraction.text import CountVectorizer
+from sklearn.feature_extraction.text import TfidfVectorizer
+from sklearn.pipeline import FeatureUnion
+from sklearn.pipeline import Pipeline
+
+from beard.applications.utils import get_author_full_name
+from beard.applications.utils import get_author_other_names
+from beard.applications.utils import get_author_initials
+from beard.applications.utils import get_author_affiliation
+from beard.applications.utils import get_title
+from beard.applications.utils import get_journal
+from beard.applications.utils import get_abstract
+from beard.applications.utils import get_coauthors
+from beard.applications.utils import get_keywords
+from beard.applications.utils import get_collaborations
+from beard.applications.utils import get_references
+from beard.applications.utils import get_year
+from beard.applications.utils import group_by_signature
+from beard.applications.utils import load_signatures
```

Comment:
Don't they pay me a dollar for every line I write?

Just kidding. If you go the other way at some point you need to break the line with import (bad), or create additional imports (inconsistent).